src/psyclone/tests/generator_test.py

```diff
@@ -175,11 +156,9 @@ def test_script_invalid_content():
         _, _ = generate(
             os.path.join(BASE_PATH, "dynamo0p3", "1_single_invoke.f90"),
             api="dynamo0.3",
-            script_name=os.path.join(
-                BASE_PATH, "dynamo0p3", "error.py"))
-    assert "attempted to import 'error' but script file " in str(error.value)
-    assert ("src/psyclone/tests/test_files/dynamo0p3/error.py' is "
-            "not valid python" in str(error.value))
+            script_name=os.path.join(BASE_PATH, "dynamo0p3", "error.py"))
+    assert ("attempted to import 'error' but a "
+            "problem was found: " in str(error.value))
```

Comment:
I'd expect to see a new test here to cover one of the situations that used to get masked. e.g. if a transformation script imported a Python module that couldn't be found?

Reply:
Done.